### PR TITLE
[IIA-49],[IIA-45] - Improved movie poster gradients and change color of the stars (on details screen)

### PR DESCRIPTION
**Pull request by evakiraly**
Simulator Screen Shot - iPhone 13 Pro Max - 2021-11-10 at 13 05 01
Simulator Screen Shot - iPhone 13 Pro Max - 2021-11-10 at 13 03 43
Simulator Screen Shot - iPhone 13 Pro Max - 2021-11-10 at 13 04 08
Simulator Screen Shot - iPhone 13 Pro Max - 2021-11-10 at 13 04 18

### CODE REVIEW

File: EnMovie/Modules/Details/DetailsViewController.swift [hunk 1]
```diff
@@ -133,13 +133,14 @@ final class DetailsViewController: UIViewController {
         verticalGradient = CAGradientLayer()
         verticalGradient.frame = imageContainerView.bounds
         verticalGradient.colors = [UIColor.clear.cgColor, UIColor.customBackgroundColor.cgColor, UIColor.customBackgroundColor.cgColor, UIColor.clear.cgColor]
-        verticalGradient.locations = [0, 0.1, 0.9, 1]
+        verticalGradient.locations = [0.03, 0.13, 0.9, 1]
```
Comment (iOSinternshipen) [inline EnMovie/Modules/Details/DetailsViewController.swift:136]: maybe move the values in a constants struct

File: EnMovie/Modules/Details/DetailsViewController.swift [hunk 2]
```diff
@@ -133,13 +133,14 @@ final class DetailsViewController: UIViewController {
...
         imageContainerView.layer.mask = verticalGradient
         
         horizontalGradient = CAGradientLayer()
+        horizontalGradient.locations = [0, 0.1, 0.9, 1]
```
Comment (iOSinternshipen) [inline EnMovie/Modules/Details/DetailsViewController.swift:140]: same here

File: EnMovie/Utils/UIColor.swift [hunk 3]
```diff
@@ -14,4 +14,5 @@ extension UIColor {
     static let customButtonColor2: UIColor = UIColor(named: "secondaryButtonBackgroundColor")!
     static let customButtonTextColor: UIColor = UIColor(named: "buttonTextColor")!
     static let tabBarItemColor: UIColor = UIColor(named: "TabBarItemColor")!
+    static let ratingViewColor: UIColor = UIColor(named: "ratingViewColor")!
```
Comment (iOSinternshipen) [inline EnMovie/Utils/UIColor.swift:17]: do you need UIColor = ? you can write directly = UIColor(named: ) 

also I don't like the force unwrap

**Review by SaniutaRobert — APPROVED** [review]

**Review by iulia-anisoi-endava — APPROVED** [review]

**Review by SaniutaRobert — APPROVED** [review]

**Review by ovidiuuvladpop — DISMISSED** [review]

**Review by iOSinternshipen — APPROVED** [review]
LGTM

File: EnMovie/Utils/UIColor.swift [hunk 4]
```diff
@@ -8,10 +8,12 @@
...
-    static let customButtonTextColor: UIColor = UIColor(named: "buttonTextColor")!
-    static let customThemeOrangeColor: UIColor = UIColor(named: "orangeThemeColor")!
+    static let customBackgroundColor = UIColor(named: "backgroundColor")!
+    static let customTextColor = UIColor(named: "textColor")!
+    static let customButtonColor1 = UIColor(named: "primaryButtonBackgroundColor")!
+    static let customButtonColor2 = UIColor(named: "secondaryButtonBackgroundColor")!
+    static let customButtonTextColor = UIColor(named: "buttonTextColor")!
+    static let tabBarItemColor = UIColor(named: "TabBarItemColor")!
+    static let ratingViewColor = UIColor(named: "ratingViewColor")!
+    static let customThemeOrangeColor = UIColor(named: "orangeThemeColor")!
```
Comment (ovidiuuvladpop) [inline EnMovie/Utils/UIColor.swift:18]: If you use force unwrap and behind the scene is nil, it will crash.
Please unwrap those optionals safely.

**Review by SaniutaRobert — APPROVED** [review]

**Pull request merged**